src/locales/de/translation.json

```
@@ -1,185 +1,185 @@
+        "card_witchermagetrophy_other": "{{count}} Hexer-/Magier-Trophäenkarten",
+        "token_level1_one": "{{count}} Rang I Plättchen",
+        "token_level1_other": "{{count}} Rang I Plättchen",
+        "token_shield_one": "{{count}} Schildmarker",
+        "token_shield_other": "{{count}} Schildmarker",
+        "underConstruction": "In Bearbeitung"
     },
     "navbar": {
-        "titleLong": "The Witcher: Old World Helper",
-        "titleShort": "Old World Helper",
+        "titleLong": "The Witcher: Die alte Welt Helfer",
+        "titleShort": "Die alte Welt Helfer",
         "communityLinks": "$t(communityLinks.title)",
-        "gameplayTools": "Gameplay Tools",
+        "gameplayTools": "Spiel-Tools",
         "setupHelper": "$t(setupHelper.title)",
         "locationTokens": "$t(locationTokens.title)",
         "monsterRoller": "$t(monsterRoller.title)",
         "lostMount": "$t(lostMount.title)",
-        "browseAllTools": "Browse All Tools",
-        "projectHelp": "Project Help",
-        "githubIssues": "Github Issues",
-        "githubDiscussions": "Github Discussions",
+        "browseAllTools": "Entdecke alle Tools",
+        "projectHelp": "Projekt-Hilfe",
+        "githubIssues": "Github Fehler",
+        "githubDiscussions": "Github Diskussionen",
         "projectTimeline": "$t(footer.projectTimeline)",
         "__flag": "🇩🇪"
     },
     "monsterPicker": {
-        "toggle": "Toggle changes:",
-        "sirenExplain": "Siren excluded as it can't be randomly selected",
-        "legendary": "Legendary",
-        "allLegendary": "All Legendary",
-        "title": "Roll A New Monster"
+        "toggle": "Umschalten bewirkt:",
+        "sirenExplain": "Sirene ausgeschlossen, da sie nicht zufällig gewählt werden kann",
+        "legendary": "Legendär",
+        "allLegendary": "Alle Legendären",
+        "title": "Würfel ein neues Monster"
     },
     "monsterRoller": {
-        "title": "Monster Roller"
+        "title": "Monster Würfeln"
     },
     "exps": {
-        "adventurePack": "Adventure Pack",
-        "lostMount": "Lost Mount",
+        "adventurePack": "Abenteuerset",
+        "lostMount": "Verlorenes Reittier",
         "ciri": "Ciri",
-        "wildHunt": "Wild Hunt",
-        "monsterTrail": "Monster Trail",
-        "mages": "Mages",
-        "legendaryHunt": "Legendary Hunt",
+        "wildHunt": "Wilde Jagd",
+        "monsterTrail": "Monsterjagd",
+        "mages": "Zauberinnen und Magier",
+        "legendaryHunt": "Legendäre Monster",
         "skellige": "Skellige",
-        "monsterPack": "Monster Pack",
+        "monsterPack": "Monster Paket",
         "mountedEredin": "$t(monsters.Mounted Eredin)"
     },
     "locationTokens": {
-        "title": "Location Tokens",
-        "mountain": "Mountain",
-        "forest": "Forest",
-        "water": "Water",
+        "title": "Ortsplättchen",
+        "Gebirge": "Mountain",
+        "Wald": "Forest",
+        "Wasser": "Water",
```

All the values on the left hand side should remain unchanged and for translations, the values on the right should be updated. The values on the left are referenced in code in the source language (EN) and are translated into the value on the right, when the language is changed.

Here's my suggested code change you can use to update this:


```suggestion
        "mountain": "Gebirge",
        "forest": "Wald",
        "water": "Wasser",
```